otherlibs/dune-rpc/private/dune_rpc_private.ml

```diff
@@ -102,6 +102,43 @@ module Loc = struct
     iso (record (both start stop)) to_ from
 end
 
+module Target = struct
+  type t =
+    | Path of string
+    | Alias of string
+    | Library of string
+    | Executables of string list
+    | Preprocess of string list
+    | Loc of Loc.t
+
+  let sexp =
```

Comment:
Not sure about Dune conventions here, but this could be named `to_sexp` to make the behavior clearer without having to read the code

Reply:
This is not strictly a `to_sexp` function (more like a bidirectional `to_sexp` and `of_sexp`). I'm not sure what we could really call it to express that, though.

Reply:
This is not strictly a `to_sexp` function (more like a bidirectional representation of both `to_sexp` and `of_sexp`). I'm not sure what we could really call it to express that, though.

Reply:
Yes, we have yet another encoding/decoding library in dune. This one has two advantages:
* It's completely bidirectional
* It offers no flexibility whatsoever for expressing your parsers (no `junk`, `peek`, etc.)